cg/meta/workflow/taxprofiler.py

```diff
@@ -49,58 +47,47 @@ def __init__(
         self.nextflow_binary_path: str = config.taxprofiler.binary_path
         self.compute_env_base: str = config.taxprofiler.compute_env
 
-    def get_nextflow_config_content(self) -> str:
+    @property
+    def sample_sheet_headers(self) -> list[str]:
+        """Headers for sample sheet."""
+        return TaxprofilerSampleSheetEntry.headers()
+
+    @property
+    def _append_params_to_nextflow_config(self) -> bool:
+        """True if parameters should be added into the nextflow config file instead of the params file."""
+        return False
+
+    @property
+    def _multiple_samples_allowed(self) -> bool:
+        """Defines whether the analysis supports multiple samples to be linked to the case."""
+        return True
```

Comment:
Out of scope. But should an analysis be sensitive to this? Ideally it should support any number of samples as input and then deal with it, right?

Reply:
Some analyses can only handle one sample and some can handle multiple samples. So yes, an analysis is sensitive to it